command/genesis/genesis.go

```diff
@@ -185,6 +185,13 @@ func setFlags(cmd *cobra.Command) {
 			command.DefaultPremineBalance,
 			"the amount which will be premined to all the validators",
 		)
+		cmd.Flags().StringVar(
+			&params.smartContractsRootPath,
+			smartContractsRootPathFlag,
```

Comment:
Right now, is this only for `polybft` right?

Reply:
I think Stefan meant that it should not be possible to see/use/anything flags which belongs to some consensus if other consensus is set by `--consensus` flag

Reply:
The thing is that you can not hide or show flags if using `--help` for example, this kind of behaviour is not very common.

Reply:
In that case, I'm suggesting leaving as it is currently if you all agree until we come up with some smarter approach?